internal/postgres/config.go

```diff
@@ -96,6 +98,71 @@ func LogDirectory() string {
 	return fmt.Sprintf("%s/logs/postgres", dataMountPath)
 }
 
+// LogRotation returns parameters that rotate log files while keeping a minimum amount.
+// Postgres truncates and reuses log files after that minimum amount.
+// Log file names start with filePrefix and end with fileSuffix.
+//
+// NOTE: These parameters do *not* enable logging to files. Set "logging_collector" for that.
+func LogRotation(minimum metav1.Duration, filePrefix, fileSuffix string) map[string]string {
+	hours := math.Ceil(minimum.Hours())
+
+	// The "log_filename" parameter is interpreted similar to `strftime`;
+	// escape percent U+0025 by doubling it.
+	// - https://www.postgresql.org/docs/current/runtime-config-logging.html#GUC-LOG-FILENAME
+	prefix := strings.ReplaceAll(filePrefix, "%", "%%")
+	suffix := strings.ReplaceAll(fileSuffix, "%", "%%")
```

Comment:
⛏️ we're doubling `%` to escape in case the prefix/suffix have `%` -- but ours don't, right?
and if someone forgot/missed that we're handling this and doubled the `%` originally, we'd quadruple the `%`?

Reply:
> ours don't, right?

Right. Escaping occurred to me as I was writing the test. I expect the `postgres` package to know about these details, while other packages don't.

> if someone forgot/missed that we're handling this and doubled the `%` originally, we'd quadruple the `%`?

Yep. If someone thinks to escape before calling, their test would show the wacky number of escapes.